src/app/components/MediaLoader/index.styles.ts

```diff
@@ -52,21 +53,38 @@ export default {
       !isEmbedded && commonMarginSpacing,
     ],
 
-  audioMediaContainer: ({ palette }: Theme) =>
+  audioMediaContainer: ({ mq, spacings }: Theme) =>
     css({
-      backgroundColor: palette.WHITE,
       height: '165px',
+      '.audio-post &&': {
+        height: '140px',
```

Comment:
This needs to be a min of 140 so that the seekbar will be displayed at all breakpoints. However, if required, we can make it 140px at smallest breakpoint & 130px above that?